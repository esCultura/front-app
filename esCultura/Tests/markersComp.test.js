import React from 'react';
import MarkersMap from '../src/components/MarkersComp';
import {render, cleanup, fireEvent} from 'react-native-testing-library';

afterEach(cleanup);

describe('<MarkersMap />', () => {

    //every it is a test
    it('should match snapshot', () => {
        const onVarChange = jest.fn();
        const comp = render(<MarkersMap  
            queryFilter={""}
            longitudeDivice = {41.389324}
            latitudeDivice = {2.113703}/>);
        let component = comp.toJSON();
        expect(component).toMatchSnapshot();
    });  
});