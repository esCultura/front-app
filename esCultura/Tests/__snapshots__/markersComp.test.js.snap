// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MarkersMap /> should match snapshot 1`] = `null`;
